### PR TITLE
All old ParksandRecreation pages redirect to wp site

### PR DESCRIPTION
**Pull request by karissademi**

### CODE REVIEW

File: rules.txt [hunk 1]
```diff
@@ -228,6 +228,8 @@
 /posts/office-of-sustainability/2017-03-28-environment-sustainability-city-of-philadelphia-action-guide 301 /2017-11-29-environmental-protection-during-the-trump-administration/
 /potholes                                 301  http://www.philadelphiastreets.com
 /ppr                                      301  /departments/philadelphia-parks-recreation/
+/ParksandRecreation/Pages/(.*)            301  /departments/philadelphia-parks-recreation/  #Handle department homepage
```
Comment (timwis) [inline rules.txt]: Hm, I think this case should be covered by the next one, shouldn't it?

Comment (karissademi) [reply rules.txt]: Lol, yeah when you're being logical about it.

File: rules.txt [hunk 2]
```diff
@@ -228,6 +228,7 @@
 /posts/office-of-sustainability/2017-03-28-environment-sustainability-city-of-philadelphia-action-guide 301 /2017-11-29-environmental-protection-during-the-trump-administration/
 /potholes                                 301  http://www.philadelphiastreets.com
 /ppr                                      301  /departments/philadelphia-parks-recreation/
+/ParksandRecreation/(.*)                  301  /departments/philadelphia-parks-recreation/
```
Comment (timwis) [inline rules.txt]: oh, also this can be lowercase because the regex rules are made case insensitive. NBD though either way. good to go!

**Comment by timwis** [comment]
I think you meant to make this PR against `staging` instead of `master`. I'll close and do a new PR against `staging`.

**Pull request closed without merging**